R/Arena.R

```diff
@@ -2599,14 +2602,17 @@ setMethod("findFeeding3", "Eval", function(object, time, mets){
     if(class(interact)=="character"){interact = t(as.matrix(interact))}
     if(nrow(interact)!=0){inter = rbind(inter,data.frame(prod=interact[,1],cons=interact[,2],met=i))}
   }
-  if(any(dim(inter)==0)) stop("No feeding found. Try other metaboites or time points.")
+ if(any(dim(inter)==0)) {ifelse(test = (tail(times,n=1)==time),no= next, 
```

Comment:
this is not a well applied ``ifelse``
> ifelse returns a value with the same shape as test
this is certainly not the case here, use if/else instead